src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
+
+  const activeTabHandler = tabId => {
+    if (tabId !== activeTab) {
+      onTabSelected(tabId);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={activeTab.id === tab.id && `is-active`}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => activeTabHandler(tab.id)}
+              >
+                {tab.title}
+              </a>
+            </li>
```

Comment:
You forgot to add a `key` prop to the `li` element inside your map function. React uses the `key` prop to optimize the rendering of lists. Each child in a list should have a unique `key` prop. In this case, you can use the `tab.id` as a unique key. Here is how you can do it: jsx
<li
 key={tab.id}
 className={activeTab.id === tab.id && `is-active`}
 data-cy="Tab"
>
```